bitnami/argo-cd/templates/applicationset/deployment.yaml

```diff
@@ -0,0 +1,183 @@
+      {{- end }}
+      containers:
+        - name: applicationset-controller
+          image: {{ include "argocd.image" . }}
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          {{- if .Values.applicationSet.command }}
+          command: {{- include "common.tplvalues.render" (dict "value" .Values.applicationSet.command "context" $) | nindent 12 }}
+          {{- end }}
+          {{- if .Values.applicationSet.args }}
+          args: {{- include "common.tplvalues.render" (dict "value" .Values.applicationSet.args "context" $) | nindent 12 }}
+          {{- else }}
+          args:
+            - argocd-applicationset-controller
+            - --logformat
+            - {{ .Values.applicationSet.logFormat }}
+            - --loglevel
+            - {{ .Values.applicationSet.logLevel }}
+            - --metrics-addr
+            - {{ .Values.applicationSet.defaultArgs.metricsAddr | quote }}
+            - --probe-addr
+            - {{ .Values.applicationSet.defaultArgs.probeBindAddr | quote }}
+            {{- if or (gt ( .Values.applicationSet.replicaCount | int64) 1) .Values.applicationSet.defaultArgs.enableLeaderElection }}
+            - --enable-leader-election=true
+            {{- end }}
+            - --argocd-repo-server
+            - {{ include "argocd.repo-server" . }}:{{ .Values.repoServer.service.port }}
+            - --policy
+            - {{ .Values.applicationSet.defaultArgs.policy | quote }}
+            - --debug
+            - {{ .Values.applicationSet.defaultArgs.debug | quote }}
+            - --dry-run
+            - {{ .Values.applicationSet.defaultArgs.dryRun | quote }}
+          {{- if .Values.applicationSet.extraArgs }}
+          {{- include "common.tplvalues.render" (dict "value" .Values.applicationSet.extraArgs "context" $) | nindent 12 }}
+          {{- end }}
+          {{- end }}
+          env:
+            {{- if .Values.applicationSet.extraEnvVars }}
+            {{- include "common.tplvalues.render" (dict "value" .Values.applicationSet.extraEnvVars "context" $) | nindent 12 }}
+            {{- end }}
+          envFrom:
+            {{- if .Values.applicationSet.extraEnvVarsCM }}
+            - configMapRef:
+                name: {{ include "common.tplvalues.render" (dict "value" .Values.applicationSet.extraEnvVarsCM "context" $) }}
+            {{- end }}
+            {{- if .Values.applicationSet.extraEnvVarsSecret }}
+            - secretRef:
+                name: {{ include "common.tplvalues.render" (dict "value" .Values.applicationSet.extraEnvVarsSecret "context" $) }}
+            {{- end }}
+          ports:
+            - name: metrics
+              containerPort: {{ .Values.applicationSet.containerPorts.metrics }}
+              protocol: TCP
+            - name: probe
+              containerPort: {{ .Values.applicationSet.containerPorts.probe }}
+              protocol: TCP
+            - name: webhook
+              containerPort: 7000
+              protocol: TCP
```

Comment:
Where are these ports configured in the argo-cd command above? If these cannot be configured, it would not make sense to allow customizing them